ducktape/services/service.py

```diff
@@ -51,11 +52,17 @@ class Service(TemplateRenderer):
     # }
     logs = {}
 
-    def __init__(self, context, num_nodes, *args, **kwargs):
+    # TODO: I changed this from num_nodes to nodes but I may not have updated all places where this constructor is called.
```

Comment:
I could also just leave `num_nodes`.  You tell me what's best here -- no change, change the param name, or bump the version to `0.7.0`. I'm flexible.